.prettierrc.json

```diff
@@ -1,15 +1,25 @@
 {
   "overrides": [
+    {
+      "files": "*",
+      "options": {
+        "printWidth": 120
+      }
+    },
     {
       "files": "*.ts",
       "options": {
-        "printWidth": 120,
         "parser": "typescript",
         "singleQuote": true,
         "bracketSpacing": false,
         "arrowParens": "avoid",
         "trailingComma": "none"
       }
+    }, {
+      "files": "*.md",
+      "options": {
+        "proseWrap": "never"
```

Comment:
"bracketSpacing": false. ?

Reply:
I fixed this by setting the same format for everything. Does this look good to you?